js/graph_config.js

```diff
@@ -535,6 +535,39 @@ GraphConfig.load = function(config) {
                                 return getCurveForMinMaxFieldsZeroOffset(fieldName);
                         }
                         break;
+                    case 'FF_LIMIT':
```

Comment:
We are changing `FF` in firmware and configurator to feedforward also.

```suggestion
                    case 'FEEDFORWARD_LIMIT':
```